src/definitions/namespaces/org_iso_18013_5_1/mod.rs

```diff
@@ -29,6 +29,7 @@ use crate::{
 
 /// The `org.iso.18013.5.1` namespace.
 #[derive(Debug, Clone, FromJson, ToCbor)]
+#[isomdl(crate = "crate")]
```

Comment:
An alternative to passing an attribute would be to infer the crate using `CARGO_PKG_NAME` env.

Here's an example:

```rust
pub(crate) fn isomdl_path_import() -> proc_macro2::Ident {
    let crate_name = std::env::var("CARGO_PKG_NAME").expect("failed to get crate name");

    if &crate_name == "isomdl" {
        proc_macro2::Ident::new("crate", proc_macro2::Span::call_site())
    } else {
        proc_macro2::Ident::new("isomdl", proc_macro2::Span::call_site())
    }
}
```

And then replace the derived input attributes with:

```rust
// Use the method instead of parsing the attributes.
let isomdl_path = isomdl_path_import();

    // let isomdl_path = Ident::new(
    //     &attrs
    //         .iter()
    //         .filter_map(super::crate_path)
    //         .next()
    //         .unwrap_or_else(|| "isomdl".to_owned()),
    //     Span::call_site(),
    // );
```

Then you can remove the attributes tag and it will infer it automatically for the user.

Reply:
This does work almost everywhere -- except in tests in `isomdl` itself. For that reason I would prefer to keep the current implementation. It also has the benefit of allowing an importer to rename the import if they happen to use an alias for isomdl in their Cargo.toml.

Reply:
Adding `::` before `isomdl` might solve this issue with the integration tests. I haven't test this though.

`proc_macro2::Ident::new("isomdl", proc_macro2::Span::call_site())` -> `proc_macro2::Ident::new("::isomdl", proc_macro2::Span::call_site())`

That said, I'm fine leaving it as it is.